### PR TITLE
add proxy name config

### PR DESCRIPTION
**Pull request by luckywaa**
## Summary
Allow the "name" section to be customizable to prevent conflicts.

## Reviewers
@huxiaoxu2019

### CODE REVIEW

File: repository.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name: FRP Client
-url: https://github.com/huxiaoxu2019/hass-addon-frp-client
+url: https://github.com/luckywaa/hass-addon-frp-client
```
Comment (huxiaoxu2019) [inline repository.yaml:2]: Please revert this change

File: frp-client/frpc.toml [hunk 2]
```diff
@@ -20,13 +20,11 @@ webServer.user = "admin"
 webServer.password = "123456789"
 
 [[proxies]]
-name = "homeassistant"
-type = "http"
-customDomains = ["your_domain"]
-transport.useEncryption = true
-transport.useCompression = true
+name = "your_proxy_name"
+type = "tcp"
```
Comment (huxiaoxu2019) [inline frp-client/frpc.toml:24]: why change to `tcp`?

File: frp-client/frpc.toml [hunk 3]
```diff
@@ -20,13 +20,11 @@ webServer.user = "admin"
...
 localPort = 8123
 localIP = "0.0.0.0"
+remotePort = 25553
```
Comment (huxiaoxu2019) [inline frp-client/frpc.toml:27]: why add `remotePort`?

File: frp-client/frpc.toml [hunk 4]
```diff
@@ -20,13 +20,11 @@ webServer.user = "admin"
 webServer.password = "123456789"
 
 [[proxies]]
-name = "homeassistant"
-type = "http"
-customDomains = ["your_domain"]
```
Comment (huxiaoxu2019) [inline frp-client/frpc.toml:25]: Would you please keep all these configurations as before, but only change the `name`? I think that's what you want in this PR.

**Comment by huxiaoxu2019** [comment]
Thanks so much for your contribution, PTAL @luckywaa

File: frp-client/frpc.toml (hunk 3 above)
Comment (luckywaa) [reply frp-client/frpc.toml:27]: 我用http协议没有办法访问，只能用tcp协议的，本来没想把最后两个commit给merge过来的。。

File: frp-client/frpc.toml (hunk 4 above)
Comment (luckywaa) [reply frp-client/frpc.toml:25]: 自己直接小改一下吧，我的sed命令也写的有问题，辛苦大佬了

File: frp-client/frpc.toml (hunk 3 above)
Comment (luckywaa) [reply frp-client/frpc.toml:27]: 用http协议会出现400错误

**Comment by huxiaoxu2019** [comment]
Merged by https://github.com/huxiaoxu2019/hass-addon-frp-client/pull/9
